.github/actions/opentelemetry/action.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: OpenTelemetry report
+description: Export GitHub actions as OpenTelemetry traces
+inputs:
+  vaultUrl:
+    description: 'Vault URL'
+    required: true
+  vaultRoleId:
+    description: 'Vault role ID'
+    required: true
+  vaultSecretId:
+    description: 'Vault secret ID'
+    required: true
+  githubToken:
+    description: 'GitHub token.'
+    required: true
```

Comment:
Thanks for this!

> githubToken could become:

+1 

> for the vault secrets we could use

Do you mean `default: {{ secrets.VAULT_ROLE_ID }}` rather than `description: {{ secrets.VAULT_ROLE_ID }}`?

If so, let's give a swing!

Reply:
ah yes :D `default`

Reply:
See https://github.com/elastic/apm-pipeline-library/pull/2009/commits/f7cbf9fd37a1e8c71f3bf625a4bf2bb979622584 and https://github.com/elastic/apm-pipeline-library/pull/2009/commits/91c1d3e15a497eb0b3f44ea298061a9a584ba676